### PR TITLE
Meeting minutes 2022-06-30

### PR DESCRIPTION
**Pull request by sthagen**
Please kindly review so we can fix any errors introduced and merge well before the next meeting.

### CODE REVIEW

File: meeting_minutes/20220930_SARIF_TC_61.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  * ONGOING - in progress, but delayed because of people on vacation
+* ACTION on Chris M. to solicit a working example from WhiteSource of their scenario
+  * ONGOING - still ongoing but the demo was already shared (tool now called Mend)
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* Chris Wysopal will obtain voting rights at the end of this meeting.
+* Pavel Mikula will obtain voting rights at the end of this meeting.
```
Comment (marmegh) [inline meeting_minutes/20220930_SARIF_TC_61.md:45]: Should these lines be under 1.7.2?

File: meeting_minutes/20220930_SARIF_TC_61.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    ```
+- Possible face-to-face meeting when pandemic permits
+
+# 3. Liaisons
+
+## 3.1 [OpenSSF](https://openssf.org/) Security Tools group
+
+> Liaison Group: Thanassis Avgerinos, Contrast Security, Microsoft
+
+* Report status and any communication from OpenSSF Scurity Tools
```
Comment (pavel-mikula-sonarsource) [inline meeting_minutes/20220930_SARIF_TC_61.md:76]:
```suggestion
* Report status and any communication from OpenSSF Security Tools
```

File: meeting_minutes/20220930_SARIF_TC_61.md (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply meeting_minutes/20220930_SARIF_TC_61.md:45]: I think they should, it was 2nd meeting in a row for both of us.

**Review by kristylee88 — APPROVED** [review]

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**